lib/hydra/works/models/concerns/generic_file/characterization.rb

```diff
@@ -0,0 +1,116 @@
+module Hydra::Works::GenericFile
+  module Characterization
+    extend ActiveSupport::Concern
+
+    autoload :FitsDatastream, 'hydra/works/models/generic_file/characterization/fits_datastream.rb'
```

Comment:
I was unable to specify that the rails autoload logic should be looking in the models directory.